frontend/websites/puconvocation-com/src/app/[locale]/(console)/console/attendees/page.tsx

```diff
@@ -10,15 +10,167 @@
+        </h1>
+        <p className={"text-xs text-gray-600"}>
+          View and manage the complete list of attendees for the convocation.
+        </p>
+      </div>
+
+      <div className={"flex min-h-screen flex-col items-center p-1"}>
+        <div className="mb-2 grid w-full grid-cols-1 gap-6 lg:grid-cols-1">
+          {/* Total Attendees */}
+          <Card className="h-fit w-full border border-gray-300 p-4">
+            <CardHeader>
+              <CardTitle>Total Attendees</CardTitle>
+              <CardDescription>
+                Showing total number of attendees for the convocation.
+              </CardDescription>
+            </CardHeader>
+            <CardContent className="flex items-center justify-between">
+              <span className="text-lg">{attendeeCount}</span>
+              <Button
+                onClick={() => setShowAttendees(!showAttendees)}
+                className="bg-red-600 text-white hover:bg-red-700"
+              >
+                {showAttendees ? "Unlock" : "Lock"}
+              </Button>
+            </CardContent>
+          </Card>
+
+          {/* Attendee List */}
+          <Card className="h-fit w-full border border-gray-300 p-4 shadow-none">
+            <CardHeader>
+              <CardTitle>Attendee List</CardTitle>
+              <CardDescription>
+                Showing detailed list of registered attendees with their
+                respective information.
+              </CardDescription>
+            </CardHeader>
+            <CardContent className="flex h-96 flex-col overflow-y-auto">
+              {isAttendeeLoading ? (
+                <div className="flex h-full items-center justify-center">
+                  <ProgressBar type="circular" />
+                </div>
+              ) : attendeeError ? (
+                <p className="text-red-600">Error loading attendees</p>
+              ) : currentAttendees.length > 0 ? (
+                <table className="min-w-full table-auto border-collapse">
+                  <thead>
+                    <tr>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Convocation Id
+                      </th>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Name
+                      </th>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Enclosure
+                      </th>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Row
+                      </th>
+                      <th className="px-4 py-2 text-center font-semibold text-gray-700">
+                        Seat
+                      </th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {currentAttendees.map((a) => (
+                      <tr
```

Comment:
Here, on clicking on any of the content row, a popup must be rendered in order to show all the details of the attendee such as department, institute and all the other fields.